build.sbt

```diff
@@ -272,6 +275,26 @@ lazy val docs = akkaModule("akka-docs")
   .disablePlugins(MimaPlugin, WhiteSourcePlugin)
   .disablePlugins(ScalafixPlugin)
 
+// shaded jctools jar
+lazy val jcTools = Project("akka-jctools", file("akka-jctools"))
```

Comment:
You could say the main innovation of this PR so far is doing the shading here automatically. One drawback is that `assembly` is now called for every compile. `assembly` does some caching but it's not super fast (can be <1s so it might not matter). Still if we depend on it in akka-actor it might too much (it also logs quite a lot which I find irritating).